src/Controls/samples/Controls.Sample/Converters/ThicknessConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Can't you just use the converter in MAUI Controls?

Reply:
The `ThicknessTypeConverter`? I think that is a `TypeConverter` and not a `IValueConverter`. Can it be used?